rest/route/stats.go

```diff
@@ -484,3 +487,51 @@ func (tsh *taskStatsHandler) Run(ctx context.Context) gimlet.Responder {
 func makeGetProjectTaskStats(sc data.Connector) gimlet.RouteHandler {
 	return &taskStatsHandler{sc: sc}
 }
+
+type cedarTestStatsMiddleware struct{}
+
+func CheckCedarTestStats() gimlet.Middleware {
```

Comment:
Does this have to be exported?

Reply:
No, but all the other middleware creation funcs are so I followed the pattern. I can make it private if you want.

Reply:
I don't see any reason to export it. I highly doubt anyone would ever use it, and I imagine the others shouldn't be exported either unless there's good reason.